packages/integration-tests/src/test-projects/data-and-function/amplify/functions/noop-import/handler.ts

```diff
@@ -0,0 +1,22 @@
+import type { Handler } from 'aws-lambda';
+import { Amplify } from 'aws-amplify';
+import { generateClient } from 'aws-amplify/data';
+import type { Schema } from '../../data/resource.js';
+import { getAmplifyDataClientConfig } from '@aws-amplify/backend/function/runtime';
+// @ts-ignore
+import { env } from '$amplify/env/noop-import.js';
+import { S3Client } from '@aws-sdk/client-s3';
+
+const { resourceConfig, libraryOptions } = await getAmplifyDataClientConfig(
+  env
+);
+
+Amplify.configure(resourceConfig, libraryOptions);
+
+const client = generateClient<Schema>();
+
+export const handler: Handler = async () => {
+  const _s3Client = new S3Client();
```

Comment:
Just for my understanding, what's the significance of this `noop-import` function?

Reply:
I added this test as a followup to conversation with @sobolk about having a function that uses `S3Client` separate from the usage in `getAmplifyDataClientConfig` that gets the MIS. 

My understanding is that with the SSM integration there was an issue that occurred when the user used SSM in their handler, which is [tested for here](https://github.com/aws-amplify/amplify-backend/blob/main/packages/integration-tests/src/test-projects/advanced-auth-and-functions/amplify/func-src/handler_with_ssm.ts)

Reply:
Got it, I'd think if the related SSM test works then it would work for other sdk clients as well, and we have this other test https://github.com/aws-amplify/amplify-backend/blob/main/packages/integration-tests/src/test-projects/advanced-auth-and-functions/amplify/func-src/handler_with_aws_sdk.ts

If there is still value for this test, I'd rename this as `customer-s3-import` or something similar to make it clear the purpose of this test.

Reply:
> I'd think if the related SSM test works then it would work for other sdk clients as well

Unfortunately, asserting that "SSM imports in banner" are in such a way that don't affect customer's SSM imports does not imply that S3 scenario works. These are two instances of similar pattern, but their "content" does not overlap, hence should be asserted separately.

Reply:
Regarding https://github.com/aws-amplify/amplify-backend/blob/main/packages/integration-tests/src/test-projects/advanced-auth-and-functions/amplify/func-src/handler_with_aws_sdk.ts .

That project was created to assert that we're bundling SDKs. 
It does not have `data` however, so can't be used to assert that "S3 client works AND data client works at the same time". Then, adding `data` there would make it overloaded.

Reply:
Perhaps this should be `AdvancedDataScenariosTestProject` where primary purpose is to dump scenarios originating from data (whether they involve other resources or not).

Reply:
My general recommendation is to start specific and go wide if/when there is need be. The next time someone adds something to this test, it would be purposely added instead of "anything data related that seems advanced to me".

Reply:
Sounds good. Let's do `DataAccessFromFunctionTestProject` then.